src/po/fr.po

```diff
@@ -4338,7 +4338,7 @@ msgid "E663: At end of changelist"
 msgstr "E663: � la fin de la liste des modifications"
 
 msgid "Type  :quit  and press <Enter> to exit Vim"
-msgstr "tapez  :q<Entr�e>  pour quitter Vim"
+msgstr "tapez  :q et alors la touche 'Entr�e'  pour quitter Vim"
```

Comment:
The French translation is wrong, or not natural.  I'd rather say:

tapez  :q  puis la touche <Entrée> pour quitter Vim

I can update the French and Esperanto translations after this change is merged into the main line anyway, along with with other unrelated translation updates.